Makefile

```diff
@@ -122,9 +122,9 @@ CUOBJ = $(patsubst %.cu, build/%_gpu.o, $(CUSRC))
 
 # extra operators
 ifneq ($(EXTRA_OPERATORS),)
-	EXTRA_SRC = $(wildcard $(patsubst %, %/*.cc %/*/*.cc, $(EXTRA_OPERATORS)))
+	EXTRA_SRC = $(wildcard $(foreach DIR, $(EXTRA_OPERATORS), $(foreach PAT, *.cc */*.cc, $(DIR)/$(PAT))))
```

Comment:
Then only the first dir in $(EXTRA_OPERATORS) will be substituted. 
Pretty awkward.